code/src/java/pcgen/gui2/doomsdaybook/NameGenPanel.java

```diff
@@ -690,7 +687,7 @@ private void loadCatalogDD()
 			join.retainAll(sexes);
 			Collections.sort(join, new DataElementComperator());
 
-			Vector<RuleSet> catalogs = new Vector<RuleSet>();
+			List<RuleSet> catalogs = new ArrayList<>();
```

Comment:
I didn't say avoid it, but if you use ArrayList, say what kind of ArrayList it is. In this case ArrayList<RuleSet>(), i.e. not as a generic.